app.py

```diff
@@ -7,15 +7,23 @@
 from flask import Flask, session
 
 app = Flask(__name__)
-app.config.update(SECRET_KEY=os.environ.get("FLASK_SECRET_KEY", "some strong secret key"))
-cache = redis.Redis(host="redis", port=6379)
+app.config.update(SECRET_KEY=os.getenv("FLASK_SECRET_KEY", "some strong secret key"))
+cache = redis.Redis(host=os.getenv("REDIS_HOST", "redis"), port=os.getenv("REDIS_PORT", 6379))
+
+
+@app.route("/ready")
+def ready():
+    try:
+        return str(cache.ping()), 200
```

Comment:
This is great as the app depends on Redis and cannot be considered "ready" until is able to connect to it